go.mod

```diff
@@ -162,3 +161,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-libs => github.com/ycombinator/elastic-agent-libs v0.0.0-20230321201000-bddf8be0aafb
```

Comment:
This line will be removed once https://github.com/elastic/elastic-agent-libs/pull/108 is merged.